vacuum-advisor/bloat.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'VACUUM Advisor: Bloat in Postgres'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+In Postgres, bloat refers to a situation that the physical size of a table on
+disk is larger than its actual data size.
+
+## Why does bloat happen?
+
+In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify.
+VACUUM allows Postgres to reclaim these dead rows and reuse that space for
+future INSERTs or UPDATEs. Autovacuum is designed to perform VACUUM regularly in
+order to efficiently reuse space. This helps ensure that INSERTs and UPDATEs can
+use existing space instead of claiming new space. This helps prevent unnecessary
+growth of tables by utilizing existing space.
+
+### Insufficient VACUUMs
+
+If VACUUM is not performed with the appropriate frequency or at the right time,
+dead rows may not be reclaimed, causing the table to grow using new disk space
+even though there is available unused space.
+
+### Inefficient VACUUMs
+
+Even when VACUUM is performed sufficiently, it's possible that dead rows still
+cannot be reclaimed. This can happen when transactions are still accessing those
+dead rows, and the xmin horizon — represents the point up to which point VACUUM
+can clean up dead rows — is held back.
+As a result, VACUUM may not reclaim any or only a few dead rows, leading to bloat.
+
+
+## Why bloat is so bad
+
+Table bloat has several negative implications, primarily affecting the
+performance of your database.
```

Comment:
It doesn't affect anything else, does it? If you have huge bloat problems, you probably have freezing problems, too, but we should cover that separately.
```suggestion
Table bloat has several negative implications for the performance of your database.
```

Reply:
I think we could say that increased disk space it also about the capacity problem. Though it's a tiny detail so I'm fine simplifying 👍

Reply:
Ah good point. Yeah, I think we can probably gloss over disk space.